src/test/java/org/jabref/logic/l10n/LocalizationParser.java

```diff
@@ -75,6 +75,14 @@
         }
     }
 
+    public static Set<LocalizationEntry> findLocalizationParametersStringsInJavaFiles(LocalizationBundleForTest type)
+            throws IOException {
+        return Files.walk(Paths.get("src/main"))
```

Comment:
You could directly use the Files.find method and pass the filter predicate as argument.